src/components/Methodology.js

```diff
@@ -107,58 +81,41 @@ const Methodology = props => {
-              target="_blank" rel="noopener noreferrer">Wikipedia</a> was used to identify those cities.</p>
+          <p>The most populous city in each state is selected. <a href="https://en.wikipedia.org/wiki/List_of_largest_cities_of_U.S._states_and_territories_by_population" target="_blank" rel="noopener noreferrer">Wikipedia</a> was used to identify those cities.</p>
 
           <h4>City Ranking</h4>
-          <p>Cities are ranked based on the map error rate which is calculated as below.</p>
-          <p>For atlas features within the subject area:</p>
-          <img className="equation" src={require('../images/formula_simple.png')} alt="equation for calculating city ranking" />
-          <p>The lower error rate a city has, the higher it is ranked. The error rate is normalized so that more weights are 
-            assigned to atlas features that are flagged multiple times.</p>
+          <p>Cities are ranked based on the map error rate with error OSM features over all OSM features. The lower error rate a city has, the higher it is ranked.</p>
+
+          <h4>OSM PBF Extraction</h4>
+          <p> The OSM data used was extracted from a full OSM history PBF file (<a href="https://planet.openstreetmap.org/">Source</a>). The goal of the initial phase of data processing was to have two extracts for each city, one containing 2018 data and the other 2019 data. This was achieved using a tool called <a href="https://osmcode.org/osmium-tool/">Osmium</a>.</p>
+          <p> The first step was to cut the full planet OSM history PBF into a history file for each city using Osmium's 'extract' tool. The tool used geojson boundaries of each city to perform a soft clip for each, generating the new city history PBFs.</p>
+          <p>Next, each of the city history files were time sliced. Using Osmium's 'time-filter' tool each history file was filtered so that the data appeared the same as it would have at a specific point in time in OSM. For each city this was done twice, creating a PBF for April 7th at 10am UTC for both 2018 and 2019. At this point, the new time slice PBFs were in standard OSM PBF format, as opposed to OSM history files.</p>
+          <p>Finally, extraneous Nodes were cleaned from the time slice PBFs using a simple python script. The Nodes in question did not have an spatial information, as they were remnants of features that had been deleted at that point in time.</p>
+
+          <h4>MQM Extent Enhancement</h4>
+          <p>An addition to MQM was incorporated into our methodology this year, to better compare differences over time. The additional feature allows the extent of MQM results to be locked to a specified bounding box, as opposed to being dynamically set by the extent of the input data. This allowed the results of two MQM runs to share the same extent despite small variances in the spatial bound of the input data.</p>
+
+          <h4>Social-Economic Score in MQM</h4>
```

Comment:
Suggest rename this section to 'Re-prioritize Map Error Hot-Spots by Usage', and combine it with the next section 'Re-weight MQM score by usage' (i.e. delete that section).